internal/descriptor/services.go

```diff
@@ -325,9 +325,6 @@ func (r *Registry) resolveFieldPath(msg *Message, path string, isPathParam bool)
 		if f == nil {
 			return nil, fmt.Errorf("no field %q found in %s", path, root.GetName())
 		}
-		if !(isPathParam || r.allowRepeatedFieldsInBody) && f.GetLabel() == descriptorpb.FieldDescriptorProto_LABEL_REPEATED {
-			return nil, fmt.Errorf("repeated field not allowed in field path: %s in %s", f.GetName(), path)
-		}
```

Comment:
Not sure this check is safe to remove, we still don't support repeated fields in paths, do we?

Reply:
Hmm, there may have been a bug in the original code then. Here was my thinking for removing it:

```go
if !(isPathParam || r.allowRepeatedFieldsInBody) && f.GetLabel() == descriptorpb.FieldDescriptorProto_LABEL_REPEATED
```

simplifies to

```go
if !(isPathParam || true) && f.GetLabel() == descriptorpb.FieldDescriptorProto_LABEL_REPEATED
```

simplifies to

```go
if false && f.GetLabel() == descriptorpb.FieldDescriptorProto_LABEL_REPEATED
```

so it would never execute the body of the `if`, so I removed it.

I can change it to this if you like:

```go
if isPathParam && f.GetLabel() == descriptorpb.FieldDescriptorProto_LABEL_REPEATED
```

Reply:
Yeah I was trying to grok what the original code was doing too. I think your simplification is correct, can you try that and see if it breaks any tests 😂?

Reply:
Heh, the tests passed with that change, so I wrote a new test based on `TestCauseErrorWithPathParam`.

Reply:
Thank you!